### PR TITLE
AWS: Support setting description for Glue table

### PR DESCRIPTION
**Pull request by lkokhreidze**
Fixes https://github.com/apache/iceberg/issues/9462

### CODE REVIEW

**Review by singhpk234 — COMMENTED** [review]
can you please also update the integ test for TestGlueCatalogTable#testCreateTable to assert the description being set ?

File: aws/src/main/java/org/apache/iceberg/aws/glue/IcebergToGlueConverter.java [hunk 1]
```diff
@@ -59,7 +60,7 @@ private IcebergToGlueConverter() {}
   private static final Pattern GLUE_DB_PATTERN = Pattern.compile("^[a-z0-9_]{1,252}$");
   private static final Pattern GLUE_TABLE_PATTERN = Pattern.compile("^[a-z0-9_]{1,255}$");
   public static final String GLUE_DB_LOCATION_KEY = "location";
-  public static final String GLUE_DB_DESCRIPTION_KEY = "comment";
+  public static final String GLUE_DESCRIPTION_KEY = "comment";
```
[the remark this answers is not in the document]
Comment (lkokhreidze) [reply aws/src/main/java/org/apache/iceberg/aws/glue/IcebergToGlueConverter.java:64]: Done

**Comment by lkokhreidze** [comment]
Thanks for the review @singhpk234! I've addressed your comments.

**Review by singhpk234 — APPROVED** [review]
LGTM, Thanks @lkokhreidze !

cc @jackye1995 @amogh-jahagirdar

**Review by amogh-jahagirdar — APPROVED** [review]
This looks good to me, thanks @lkokhreidze ! Thanks @singhpk234 for the review!

**Comment by amogh-jahagirdar** [comment]
@lkokhreidze Could you run spotless `./.gradlew spotlessApply` and push again? The spotless checks are failing.

**Comment by lkokhreidze** [comment]
Thanks @amogh-jahagirdar and apologies, should have done it beforehand. Pushed the changes

**Comment by amogh-jahagirdar** [comment]
No problem at all @lkokhreidze thanks for the contribution, thanks @singhpk234 for the review!

**Pull request merged**